benchmark/parallel/egonets.jl

```diff
@@ -0,0 +1,63 @@
+using LightGraphs
```

Comment:
My intension with this benchmark is to get us to see the parallel performance and track it over time. Not really a hard performance regression detector. 

Of course, if the parallel functions were slower than the serial performance, then I would call that a regression.